lib/sentry/client.ex

```diff
@@ -285,20 +285,24 @@ defmodule Sentry.Client do
   end
 
   defp render_exception(%Interfaces.Exception{} = exception) do
-    exception
-    |> Map.from_struct()
+    case Map.from_struct(exception) do
```

Comment:
Ok I missed that this was done in two places. Let's get this `case` code and extract it into a `render_stacktrace` function then!

Reply:
Done!